types/is-git-clean/index.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for is-git-clean 1.1
+// Project: https://github.com/jamestalmage/is-git-clean
+// Definitions by: Michaël De Boey <https://github.com/MichaelDeBoey>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function isGitClean(dir?: string, options?: isGitClean.Options): Promise<boolean>;
```

Comment:
From the code, it looks like you might also want an overload that accepts `options` without `dir`.

Reply:
@amcasey That's not what is stated in the [Usage](https://github.com/jamestalmage/is-git-clean#usage) or the code.
https://github.com/jamestalmage/is-git-clean/blob/e27460f0c1c6e786b9634b059c135ce59ac35495/index.js#L31-L56

Do you see something else as I am?

Reply:
I was looking at this: https://github.com/jamestalmage/is-git-clean/blob/e27460f0c1c6e786b9634b059c135ce59ac35495/index.js#L22

Having said that, I've never encountered this package before and have no knowledge of how it's used in practice.

Reply:
@amcasey It's used in both [`react-codemod`](https://github.com/reactjs/react-codemod) & [`@next/codemod`](https://github.com/vercel/next.js/tree/canary/packages/next-codemod) in exactly the same way.

https://github.com/reactjs/react-codemod/blob/b34b92a1f0b8ad333efe5effb50d17d46d66588b/bin/cli.js#L26
https://github.com/vercel/next.js/blob/canary/packages/next-codemod/bin/cli.ts#L26

Since the usage docs aren't mentioning the case you're talking about, I think it's fine to leave it as-is.
If somebody ever wants to add support, they can still add it to the types, no?

Reply:
Yep, easy to add later.  Thanks for following up!